### PR TITLE
CP-307: fix profile post page

### PR DESCRIPTION
**Pull request by cw1997**
add post status filter select box to post tab of profile page ( /u/[username]/post/[[...status]] )

### CODE REVIEW

**Comment by 634750802** [comment]
Please rebase main @cw1997

File: .eslintrc [hunk 1]
```diff
@@ -17,6 +17,6 @@
     "react/jsx-no-target-blank": [2, { "allowReferrer": true }],
     "@typescript-eslint/ban-ts-comment": "off",
     "@typescript-eslint/no-empty-function": "off",
-    "@typescript-eslint/explicit-module-boundary-types": "off"
```
Comment (634750802) [inline .eslintrc:20]: ptal

File: src/pages/blog/user/[id]/index.page.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+interface IProps {
+}
+
+interface IQuery extends ParsedUrlQuery {
+  id: string;
+}
+
+export const getStaticProps: GetStaticProps<IProps, IQuery>  = (ctx) => {
+  const { id: userId } = ctx.params;
```
Comment (634750802) [inline src/pages/blog/user/[id]/index.page.tsx:13]: we need i18n props

Comment (cw1997) [reply src/pages/blog/user/[id]/index.page.tsx:13]: OK, I will add them ASAP

**Pull request merged**